### ISSUE TITLE
[a11y] Bad contrast in bookmarklet

### ISSUE THREAD

**Issue by iw0nderhow** [nextcloud/bookmarks#2028]
**Is your feature request related to a problem? Please describe.**
The contrast between text and background is not very good in the bookmarklet view. See below for a screenshot. With the default "Clouds" background, the contrast ratio is [3.81:1](https://www.whocanuse.com/?bg=0082c8&fg=222222&fs=15&fw=) and fails WCAG2 guidelines.

**Describe the solution you'd like**
Setting a white background (ref. the Files app) for the bookmarklet should help this.

**Describe alternatives you've considered**
None yet.

**Additional context**
<img src="https://github.com/nextcloud/bookmarks/assets/6024426/e142117c-9de9-408f-b74d-36a80b30696d" width="40%">
